starfish/test/pipeline/test_filter.py

```diff
@@ -31,3 +32,22 @@ def test_gaussian_high_pass_3d(sigma: Union[Number, Tuple[Number]]) -> None:
     ghp = gaussian_high_pass.GaussianHighPass(sigma=sigma, is_volume=True)
     result = ghp.run(image_stack)
     assert np.sum(result.numpy_array) < sum_before
+
+@pytest.mark.parametrize('size', (1, (1, 1)))
```

Comment:
I think we could probably combine these since they have the same test in the end. @ttung, we might be able to apply this to other filters. 

```python3
@pytest.mark.parametrize('size, is_volume', [
    (1, False), 
    ((1, 1), True),
    ((1, 0, 1), False)
])
def test_mean_high_pass(size: Union[Number, Tuple[Number]], is_volume: bool) -> None:
...
```

Reply:
Sounds good. I'll open another PR since it will affect other filters.